src/main/scripts/canonicalize.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c), InterSociety Digital Cinema Forum (ISDCF) <info@isdcf.com>
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following 
+conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following 
+disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following 
+disclaimer in the documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/* Canonicalize the registries */
+
+require('./validate').registries().then(regs => {
+  const fs = require('fs');
+
+  for (var reg_name in regs) {
+    console.log("Canonicalizing " + regs[reg_name].name + " registry");
+    data = fs.writeFileSync(
```

Comment:
```suggestion
    fs.writeFileSync(
```

[writeFileSync return undefined](https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options).